service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/OrderIssuedEmailContentProvider.java

```diff
@@ -85,4 +115,16 @@ public OrderIssuedEmailContentProvider(@Value("${ccd.ui.base.url}") String uiBas
 
         return url.build();
     }
+
+    private Map<String, Object> linkToAttachedDocument(final byte[] documentContents) {
+        ImmutableMap.Builder<String, Object> url = ImmutableMap.builder();
+
+        try {
+            url.put("documentLink", prepareUpload(documentContents));
+        } catch (NotificationClientException e) {
+            log.error("Unable to send notification due to ", e);
```

Comment:
is this logged error correct?

Reply:
I think so, I used the same from buildCMODocumentLinkNotificationParameters(), it is doing the same check

Reply:
well, it is not correct in a way, that it says unable to send, but it does not prevent anything

Reply:
can update error message to be more accurate